bundles/org.osgi.framework.system.packages.extra/META-INF/MANIFEST.MF

```diff
@@ -4,4 +4,3 @@ Bundle-Version: 1.0.0.qualifier
 Bundle-Name: Osgi Framework Extender
 Fragment-Host: system.bundle; extension:=framework
 Bundle-SymbolicName: org.osgi.framework.system.packages.extra
-Export-Package: com.sun.source.tree,com.sun.source.util
```

Comment:
Wieso? Bitte diese Änderung genau begründen!

Reply:
Hier findet man überhaupt keine Dateien.

Damit werden folgende Fehler in der IDE-View Problems vermieden:
Package 'com.sun.source.tree' does not exist in this plug-in	MANIFEST.MF	/org.osgi.framework.system.packages.extra/META-INF	line 7	Plug-in Problem

Wahrscheinlich könnte man auch das ganze Projekt löschen, aber es wurde von Dir im Rahmen von "  [11355] Compatibility to Java >8, Target update to 2018-09 (#296)" aufgenommen.

Reply:
Hmm ja, ich kann mich erinnern das da irgenwie packages für OSGI freigegeben werden. Nimm diesen Patch
aber bitte raus, der Server läuft mit Java 11 und ich glaube das würde die Funktionalität brechen!